addons/web/static/src/js/fields/field_registry.js

```diff
@@ -91,3 +91,22 @@ registry
     .add('timezone_mismatch', special_fields.FieldTimezoneMismatch)
     .add('report_layout', special_fields.FieldReportLayout);
 });
+
+odoo.define('web.owl_field_registry', function (require) {
```

Comment:
I get that separation is good.
But this makes yet another stuff prefixed with owl. and yet another thing to prepare retro-compatibility for (partners should be able to continue pushing in the right registry, even if its name has changed)

Reply:
@kebeclibre 
What I can do is to remove this module(`web.owl_field_registry`) to not have an another registry and I move the module `web._owl_field_registry` in an other file. Like this we'll have this file which creates a registry and registers the legacy fields and an other file which registers the owl fields in the same registry (they will override the legacy ones).
And then if we only want legacy fields, we just remove the file that registers owl fields.
What do you think?